index.html

```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
             </div>
         </div>
     </section>
+    <section class="works-section">
+        <article class="div-works">
+            <img class="article_img" src="assets/images/nature35.svg" alt="Product 1">
+            <div class="article_text">
+                <h2 class="product-title">Tonic</h2>
+                <ul class="type_dev">
+                    <li class="li-color">CANOPY</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="assets/images/Counter.svg" alt="Counter image"></li>
+                    <li>2015</li>
+                </ul>
+                <p class="product-details">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
```

Comment:
- [ ] please ensure you use the appropriate `color` which is `#344563` to style this highlighted code. It is important to stick to the figma design in relation to color and fonts as this is good practice and shows how much attention to details you pay as a skilled frontend developer :+1: :sparkles: 
- [ ] Please kindly make this change in every card for the work section so your overall design matches the figma design completely.